pkgs/applications/misc/mepo/default.nix

```
@@ -53,7 +53,7 @@ stdenv.mkDerivation rec {
   installPhase = ''
     runHook preInstall
 
-    zig build -Drelease-safe=true -Dcpu=baseline --prefix $out install
+    zig build -Drelease-safe=true --prefix $out install
```

"install" is the default build step and can therefore be omitted, having the same effect as before:

```suggestion
    zig build -Drelease-safe=true -Dcpu=baseline --prefix $out
```